README.md

```diff
@@ -0,0 +1,254 @@
+![NGINX Banner](docs/img/logo.png "NGINX Banner")
+
+NGINX (pronounced "engine x" or "en-jin-eks") is the world's most popular Web Server, high performance Load Balancer, Reverse Proxy, API Gateway and Content Cache.
```

Comment:
This is a wonderful opportunity to rework the official description of what the NGINX project is.

How do we order and mention the primary capabilities? What about TCP, UDP and email protocol proxying?

I have a parallel effort to revise this description for the nginx.org website so very happy to collaborate on this.